addons/godot_tours/core/overlays/overlays.gd

```diff
@@ -13,8 +13,9 @@ const HighlightPackedScene := preload("highlight/highlight.tscn")
 const DimmerPackedScene := preload("dimmer/dimmer.tscn")
 const OverlayPackedScene := preload("overlay.tscn")
 
-const OVERLAY := "overlay"
-const HIGHLIGHT := "highlight"
+const DIMMER := "Dimmer"
+const OVERLAY: StringName = "overlay"
+const HIGHLIGHT: StringName = "highlight"
```

Comment:
When you get closer to done, could you add docstrings to things like these, to know why they're here/needed? To not forget why we have certain implementation details when revisiting this in several months